salt/client/ssh/client.py

```diff
@@ -196,3 +197,41 @@ def cmd_async(self, low, timeout=None):
         '''
         # TODO Not implemented
         raise SaltClientError
+
+    def cmd_subset(
+            self,
+            tgt,
+            fun,
+            arg=(),
+            timeout=None,
+            tgt_type='glob',
+            ret='',
+            kwarg=None,
+            sub=3,
+            **kwargs):
+        '''
+        Execute a single command via the salt-ssh subsystem on a subset of the targeted minions
```

Comment:
This docstring doesn't sufficiently describe the behavior of this function IMHO. It should be expanded to explain how those minions are selected.